elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -59,6 +60,8 @@
     private static final String SINGLE_PARAMETER_ONLY = "There can only be a single filter query parameter";
     private static final String INVALID_QUERY_PARAMETER = "Invalid query parameter: ";
     private static final Pattern TYPED_FILTER_PATTERN = Pattern.compile("filter\\[([^\\]]+)\\]");
+    private static final ComparisonOperator INI = new ComparisonOperator("=ini=", true);
+    private static final ComparisonOperator NOT_INI = new ComparisonOperator("=outi=", true);
```

Comment:
Looks good, but we should add all the insensitive operators here as well (PREFIXI, INFIXI, etc)